checklist.R

```diff
@@ -1130,7 +1130,7 @@ col_entry_data_type_sor_mismatch <- function(cohort, site, report, output_format
   output <- c()
   
   # read sor and upload file
-  sor <- get_data(config$synapse$sor$id, sheet = 2)
+  sor <- get_data(config$synapse$sor$id, sheet = 3)
```

Comment:
There was an error when i ran this code initially that looked for the presence of `VARNAME` in the scope of release file (sor in the code) but it wasn't found. When I printed out `sor` in the code, it gave a table that was pretty empty minus some irrelevant lines so my guess was that the `sor` file wasn't being read in correctly. Tracking down that the SOR file is syn22294851, I just did trial and error with the sheet numbers and sheet 3 seems to match the data dictionary sheet in the `sor` which has the expected `VARNAME` variable. Looking at other pieces of the code that uses `sor`, the data dictionary sheet seems to match what those sections of code are looking for (the other sheets don't)

Actually maybe a better fix is to have it read it in by the sheet name instead of the number.

Reply:
Lets switch from the index to "Data Dictionary"

Reply:
Hmm... theres too much discrepancy.  We will need to set up a meeting with Xindi to discuss the results.

Reply:
Sounds good, I'm having trouble getting RStudio set up for in depth troubleshooting even though I somehow did it before so wasn't able to look at the differences more closely.

Command used:
`docker run --rm genie-bpc-quac Rscript genie-bpc-quac.R  -c NSCLC -s all -r table -l error -v -a $SYNAPSE_TOKEN`

Reply:
Thanks, do you have the results of each of the different runs given the difference in sheet naming?  We can talk about all of this in the 3x meeting and get some feedback

Reply:
Unfortunately, that's what I wasn't able to find _yet_, the `nsclc_all_table_error.csv` that gets outputted as a result of running the script. I wasn't able to find it on the ec2 instance i was running this in. This is the screen that gets printed out when we use `sheet = 3`. 
<img width="913" alt="image" src="https://github.com/Sage-Bionetworks/genie-bpc-quac/assets/26471741/568d6000-661b-4ae5-a449-0a4ad172427d">

I don't have the screen shot for when we use `sheet = "Data Dictionary"`, but instead of 0 errors, there's a few hundred to thousand errors that gets written to `nsclc_all_table_error.csv`. 

And then this was when I kept failing on getting a RStudio session to look at the actual object that gets written to `nsclc_all_table_error.csv` or if the scope of release file is actually getting read in correctly with `sheet = "Data Dictionary"`...